bee-common/bee-common/tests/shutdown_stream.rs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+use bee_common::shutdown_stream::ShutdownStream;
+
+use async_std::task;
+
```

Comment:
Remove newline.

For consistency here is what we decided in the past:
4 blocks of use with one new line between, and no new line within:
- super/crate use
- bee use
- 3rd parties use
- std use

Reply:
It would be cool if rustfmt could enforce it :P

Reply:
I enforce it :trollface:

Reply:
`cargo thibfmt --deny pedantic`